nvme-ioctl.c

```diff
@@ -269,6 +269,56 @@ struct nvme_dsm_range *nvme_setup_dsm_range(__u32 *ctx_attrs, __u32 *llbas,
 	return dsm;
 }
 
+int nvme_copy(int fd, __u32 nsid, struct nvme_copy_range *copy, __u64 sdlba,
+		__u16 nr, __u8 prinfor, __u8 prinfow, __u8 dtype, __u16 dspec,
+		__u8 format, int lr, int fua, __u32 ilbrt, __u16 lbatm,
+		__u16 lbat)
+{
+	__u32 cdw12 = (nr - 1) & 0xff;
+
+	cdw12 |= (format  & 0xf) <<  8;
+	cdw12 |= (prinfor & 0xf) << 12;
+	cdw12 |= (dtype   & 0xf) << 20;
+	cdw12 |= (prinfow & 0xf) << 26;
+	cdw12 |= (fua     & 0x1) << 30;
+	cdw12 |= (lr      & 0x1) << 31;
```

Comment:
We're still using compilers that throw a warning for mixed declarations and code

Reply:
Fixed!